### PR TITLE
Updated README file

### PR DESCRIPTION
**Pull request by aquamatt**
In installing djangopypi for the first time I found that the README was not sufficiently complete for new users. Whilst I could see what was required get get package upload and download working, it could slow down others or cause them to give up. 

I have therefore added to the README file in a way that I think helps.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -46,6 +46,76 @@ Then add an include in your url config for ``djangopypi.urls``::
...
+``<MEDIA_ROOT>/dists`` directory is created and writable by the web server.
+
+You may change the directory to which packages are uploaded by setting
+``DJANGOPYPI_RELEASE_UPLOAD_TO``; this will be a sub-directory of ``MEDIA_ROOT``.
+
+
+Other settings
+++++++++++++++
+
+Look in the ``djangopy`` source code for ``settings.py`` to see other
```
Comment (msabramo) [inline README.rst]: djangopy -> djangopypi

**Comment by msabramo** [comment]
I think that this PR is fricking awesome. I also thought that the README could use a bit more detail and was thinking about adding to it, but you beat me to it! :-) I hope this is merged. Well done!

**Comment by aquamatt** [comment]
Thank you msabramo for taking the time to review and comment!

**Comment by msabramo** [comment]
No problem! If you want to review one of my PRs, check out #47 ;-)

**Comment by aquamatt** [comment]
REPLACED BY PR #53 https://github.com/benliles/djangopypi/pull/53

**Pull request closed without merging**